aospy/test/test_calc_basic.py

```diff
@@ -160,6 +161,53 @@ def setUp(self):
         }
 
 
+test_params = {
+        'proj': example_proj,
+        'model': example_model,
+        'run': example_run,
+        'var': precip_largescale,
+        'date_range': 'default',
+        'intvl_out': 1,
+}
+
+
+@pytest.mark.parametrize(
+    ('dtype_out_time'),
+    [(None),
+     ('None'),
+     ([])]
+)
+def test_calc_object_no_time_options(dtype_out_time):
+    test_params['dtype_out_time'] = dtype_out_time
+    CalcInterface(**test_params)
+
+
+@pytest.mark.parametrize(
+    ('dtype_out_time'),
+    [('av'),
+     ('std'),
+     ('ts'),
+     ('reg.av'),
+     ('reg.std'),
+     ('reg.ts')]
+)
+def test_calc_object_string_time_options(dtype_out_time):
+    test_params['dtype_out_time'] = dtype_out_time
+    with pytest.raises(ValueError):
+        CalcInterface(**test_params)
+
+
+@pytest.mark.fixture
```

Comment:
The fixture decorator shouldn't be used on a test; instead it's used to prepare the data needed for a test.  See https://docs.pytest.org/en/latest/fixture.html#fixtures